src/frontend/delete-jira-configuration.ts

```diff
@@ -9,23 +7,26 @@ import { Request, Response } from "express";
  */
 export default async (req: Request, res: Response): Promise<void> => {
 	const jiraHost = req.session.jiraHost;
+	if(!jiraHost) {
+		req.log.warn("Missing Jira Host");
+		return;
+	}
```

Comment:
Any special reason that prompted you to add this warning?

Reply:
without it, the code would fail if the jira host is missing.  There isn't any point of moving forward without it and figured we'd want to know if this is happening consistently.

Reply:
Why not throwing error? Otherwise it will return 2xx